### PR TITLE
(DOCSP-11754): Add encrypt a realm page

### PR DESCRIPTION
**Pull request by cbush**
## Pull Request Info
Adds the encryption pages based on https://realm.io/docs/swift/latest/#encryption

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-11754

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/encryption/ios/encrypt.html
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/encryption/android/encrypt.html
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/encryption/node/encrypt.html
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/encryption/react-native/encrypt.html

### CODE REVIEW

File: source/ios/encrypt.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can encrypt a :ref:`synced realm <ios-sync-data>`. {+service+} only
+encrypts the data on the device and stores the data unencrypted in your
+{+atlas+} :term:`data source`.
+
+Procedure
+---------
+
+The following code demonstrates how to open a {+realm+} with a generated
```
Comment (nlarew) [inline source/ios/encrypt.txt:48]: nit: generated by Realm or by the user?

File: source/ios/encrypt.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Encryption and Realm Sync
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can encrypt a :ref:`synced realm <ios-sync-data>`. {+service+} only
+encrypts the data on the device and stores the data unencrypted in your
+{+atlas+} :term:`data source`.
+
+Procedure
+---------
```
Comment (nlarew) [inline source/ios/encrypt.txt:46]: Not sure that we should call this a Procedure given that it doesn't use steps. Maybe either:
- turn this into a stepfile procedure (IMO ideal but scope creep from porting content)
- switch up the section title (e.g. "Example")
- accept inconsistency and move on

**Review by nlarew — APPROVED** [review]
LGTM % nits for the port

File: source/ios/encrypt.txt (hunk 2 above)
Comment (cbush) [reply source/ios/encrypt.txt:46]: Cool story: I had "Example" and thought you'd request "Procedure". I can change it back.

File: source/ios/encrypt.txt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+      .. literalinclude:: /examples/Encrypt/encrypt.m
+         :language: objective-c
+
+You must pass the same encryption key when opening the {+realm+} again.
+Apps should store the encryption key in the Keychain. The following
+Swift example demonstrates how to do this:
+
+.. literalinclude:: /examples/Encrypt/StoreInKeychain.swift
+   :language: swift
```
Comment (nlarew) [inline source/ios/encrypt.txt:70]: This seems related to the **Storing & Reusing Keys** consideration on the other pages. Should we move this to a consideration for consistency?

Comment (cbush) [reply source/ios/encrypt.txt:70]: The others didn't have a corresponding example I could develop, so this section was dangling at the end. I could move the text up and keep the example at the bottom.
